src/apps/loyalty/server/middlewares/user.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from "express"
+import { fetchCollectorProfile } from "../gravity"
+
+export default (req: Request, res: Response, next: NextFunction) => {
```

Comment:
yea, I hear you on that. I think that makes total sense 👍 , sorry for the noise.